### PR TITLE
Adding contamination estimation to the RNA pipeline

### PR DESCRIPTION
**Pull request by takutosato**
I still need to update the part of the wdl that sends output to TDR.

An analysis of RNA contamination here: https://github.com/broadinstitute/hydro.gen/issues/890

### CODE REVIEW

File: pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl
Comment (kayleemathews) [inline pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl:74]:
```suggestion
    population_vcf: "VCF file for contamination estimation containing common SNP sites from population-wide studies like ExAC or gnomAD"
```

File: pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl
Comment (kayleemathews) [inline pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl:75]:
```suggestion
    population_vcf_index: "Index file for the population vcf"
```

File: pipelines/broad/internal/rna_seq/BroadInternalRNAWithUMIs.wdl
Comment (kayleemathews) [inline pipelines/broad/internal/rna_seq/BroadInternalRNAWithUMIs.wdl:69]:
```suggestion
    tdr_dataset_uuid: "Optional string used to define the Terra Data Repo (TDR) dataset to which outputs will be ingested"
    tdr_sample_id: "Optional string used to identify the sample being processed; this is the primary key in the TDR dataset"
    tdr_staging_bucket: "Optional string defining the GCS bucket to use to stage files for loading to TDR; the workspace bucket is recommended"
```

**Comment by takutosato** [comment]
@mmorgantaylor It says merging is blocked---maybe because it needs reviewers? I requested a review from you and @gbggrant. Or, if the merge button is not grayed out for you please free to just merge it too.

**Review by mmorgantaylor — APPROVED** [review]
the TDR parts look good to me - would be great to get George's thumb too for the contamination bit

**Review by gbggrant — CHANGES_REQUESTED** [review]
Looks good. I need to add the contamination VCFs to the test files for the 'RNAWithUMIsPIpeline' - will do that shortly (so that the tests will pass)

**Comment by gbggrant** [comment]
add to whitelist

**Comment by takutosato** [comment]
Thank you @gbggrant and @mmorgantaylor

**Pull request merged**
